pkgs/os-specific/linux/audit/default.nix

```diff
@@ -23,6 +24,19 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-v0ItQSard6kqTDrDneVHPyeNw941ck0lGKSMe+FdVNg=";
   };
 
+  patches = lib.optionals (!stdenv.hostPlatform.isGnu) [
```

Comment:
Won't this result in an empty list for GNU platforms, causing a lot of rebuilds?

Reply:
Yes! Thanks for noticing.

Reply:
Actually it shouldn't, because patches defaults to `[]` in make-derivation.nix.

Reply:
We can wait for OfBorg to check though.

Reply:
rebuild-linux: 1-10